### PR TITLE
build_appbundle_test MockProcessManager -> FakeProcessManager

### PR DESCRIPTION
**Pull request by jmagman**
Replace mockito `MockProcessManager` with `FakeProcessManager` in build_appbundle_test.

Part of https://github.com/flutter/flutter/issues/71511

### CODE REVIEW

File: packages/flutter_tools/test/commands.shard/permeable/build_appbundle_test.dart [hunk 1]
```diff
@@ -201,125 +177,104 @@ void main() {
...
-        ],
-        workingDirectory: anyNamed('workingDirectory'),
-        environment: anyNamed('environment'),
-      )).thenAnswer((_) {
-        return Future<Process>.value(
-          createMockProcess(
-            exitCode: 0,
-            stdout: '',
-          ),
+    group('AndroidX', () {
```
Comment (jmagman) [inline packages/flutter_tools/test/commands.shard/permeable/build_appbundle_test.dart:184]: This section is easier to see with Hide Whitespace Changes on.

**Review by dnfield — APPROVED** [review]
I see less mockito and I approve.

**Comment by dnfield** [comment]
Once tests pass anyway

**Comment by jmagman** [comment]
It's passing on my machine, ruh roh.

**Comment by jmagman** [comment]
Closing, I will look at this some other time.

**Pull request closed without merging**